pkgs/development/libraries/qt-6/modules/qtbase.nix

```diff
@@ -152,16 +163,19 @@ stdenv.mkDerivation rec {
     xorg.libXtst
     xorg.xcbutilcursor
     libepoxy
-  ] ++ (with unixODBCDrivers; [
```

Comment:
Why did you remove this option?

Reply:
Because `with unixODBCDrivers; [ sqlite ]` erroneously evaluates to `sqlite` instead.